lxd/network_acls.go

```diff
@@ -212,7 +215,7 @@ func networkACLsPost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
-	_, err = acl.LoadByName(d.State(), projectName, req.Name)
+	netACL, err := acl.LoadByName(d.State(), projectName, req.Name)
```

Comment:
This won't work. `netACL` will be `nil` unless it already exists and if it already exists, we return early with an error.
You're then passing `netACL` as `nil` to `NetworkACLCreated` which then panics :)

Reply:
Ahhhh one of my major gripes with Go sometimes is the easy-to-miss, yet colossal difference between `err == nil` and `err != nil` :)

Reply:
Fixed it by getting the struct again after creating it. Then had to sort out omitting the ACL name in the source by checking if it's the `Created` event in the event handler.

Reply:
Actually, I just realized I don't think I do that consistently. I'll make another PR where I add that conditional in all the lifecycle handlers.